code-of-conduct.md

```diff
@@ -0,0 +1,46 @@
+# Dallas R User Group - Code of Conduct
+
+The Dallas R User Group is dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form.
+
+This code of conduct applies to all Dallas R User group spaces, including meetups, Twitter, Slack, mailing lists, both online and offline. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the Leadership Team.
+
+Participants are responsible for knowing and abiding by these rules.
+
+**Community Guidelines:**
+
+* **Be friendly and patient**: Remember you might not be communicating in someone else's primary spoken or programming language, and others may not have your level of understanding.
+* **Be welcoming**: Our communities welcome and support people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+Harassment includes:
+* **Be respectful:** We are a world-wide community of professionals, and we conduct ourselves professionally. Disagreement is no excuse for poor behavior and poor manners. 
+
+**Disrespectful and unacceptable behavior includes, but is not limited to:**
+* Violent threats or language.
+* Discriminatory or derogatory jokes and language.
+* Posting sexually explicit or violent material.
+* Posting, or threatening to post, people's personally identifying information ("doxing").
+* Insults, especially those using discriminatory terms or slurs.
+* Behavior that could be perceived as sexual attention.
+* Advocating for or encouraging any of the above behaviors.
+* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, or religion.
+* Unwelcome comments regarding a person's lifestyle choices and practices, including those related to food, health, parenting, drugs, and employment.
+* Deliberate misgendering or use of 'dead' or rejected names.
+* Gratuitous or off-topic sexual images or behaviour in spaces where they're not appropriate.
```

Comment:
IMO, I would keep `Posting sexually explicit or violent material` above, it is more general